src/express.re

```diff
@@ -36,6 +36,22 @@ module Request = {
   external baseUrl: t => string = "" [@@bs.get];
   /** [baseUrl request] returns the 'baseUrl' property */
 
+  external cookies : t => option (Js.Dict.t Js.Json.t) = "" 
```

Comment:
given that these are so tightly bound to the CookieParser module, would it make sense to have them be methods inside it, even though they effectively operate on the mutated request?